src/legacy/core_plugins/tile_map/public/tile_map_visualization.js

```diff
@@ -158,12 +159,14 @@ export const createTileMapVisualization = ({ serviceSettings, $injector }) => {
 
     _getGeohashOptions() {
       const newParams = this._getMapsParams();
-      const metricAgg = this._getMetricAgg();
-      const boundTooltipFormatter = tooltipFormatter.bind(null, this.vis.getAggConfig(), metricAgg);
+      const metricDimension = this._params.dimensions.metric;
+      const metricLabel = metricDimension ? metricDimension.label : '';
+      const metricFormat = metricDimension ? getFormat(metricDimension.format) : (x) => x;
+      const boundTooltipFormatter = tooltipFormatter.bind(null, metricLabel, metricFormat.getConverterFor('text'));
 
       return {
-        label: metricAgg ? metricAgg.makeLabel() : '',
-        valueFormatter: this._geoJsonFeatureCollectionAndMeta ? (metricAgg && metricAgg.fieldFormatter()) : null,
+        label: metricLabel,
```

Comment:
When I create a new TileMap visualization I see `undefined` instead of 'Count' here. Is it ok? 
<img width="845" alt="Screen Shot 2019-10-21 at 7 00 42 PM" src="https://user-images.githubusercontent.com/20072247/67222248-96165200-f435-11e9-9b79-4ba9ad570697.png">

Reply:
i need to double check, thanks for noting!